generated/rust-analyzer.nix

```
@@ -1985,6 +1985,133 @@
       kind = "boolean";
     };
   };
+  "rust-analyzer.workspace.discoverConfig" = {
+    description = ''
+      Enables automatic discovery of projects using
+      \[`DiscoverWorkspaceConfig::command`\].
+
+      \[`DiscoverWorkspaceConfig`\] also requires setting `progress_label` and
+      `files_to_watch`. `progress_label` is used for the title in progress
+      indicators, whereas `files_to_watch` is used to determine which build
+      system-specific files should be watched in order to reload
+      rust-analyzer.
+
+      Below is an example of a valid configuration:
+
+      ``` json
+      "rust-analyzer.workspace.discoverConfig": {
+              "command": [
+                      "rust-project",
+                      "develop-json",
+                      {arg}
+              ],
+              "progressLabel": "rust-analyzer",
+              "filesToWatch": [
+                      "BUCK",
+              ],
+      }
+      ```
+
+      *On `DiscoverWorkspaceConfig::command`*
```

I think bold would be better than italic for a header
```suggestion
      **On `DiscoverWorkspaceConfig::command`**
```

---

fixed (and in the lua filter)